domain/src/main/kotlin/io/github/wykopmobilny/domain/settings/GetAppearancePreferencesQuery.kt

```diff
@@ -62,13 +56,13 @@ class GetAppearancePreferencesQuery @Inject internal constructor(
     private fun appearanceFlow(): Flow<AppearanceSectionUi> =
         getAppearanceSectionPreferences().map { appearance ->
             AppearanceSectionUi(
-                useDarkTheme = Setting(
-                    currentValue = appearance.isDarkTheme,
-                    onClicked = { updateUserSetting(UserSettings.darkTheme, !appearance.isDarkTheme) },
+                userThemeSetting = ListSetting(
+                    values = AppThemeUi.values().toList(),
+                    currentValue = appearance.appTheme.toUi(),
+                    onSelected = { updateUserSetting(UserSettings.appTheme, it.toDomain()) },
                 ),
                 useAmoledTheme = Setting(
                     currentValue = appearance.isAmoledTheme,
-                    isEnabled = appearance.isDarkTheme,
```

Comment:
❔ 
How about keeping the `isEnabled` but in a form of:
```
                    isEnabled = when (appearance.appTheme) {
                        SavedAppTheme.Auto,
                        SavedAppTheme.Dark,
                        -> true
                        SavedAppTheme.Light -> false
                    },
```
My motivation here was to clearly state this specific option doesn't have any impact on the app state if Light mode is selected